src/button/styles/button.css

```diff
@@ -0,0 +1,47 @@
+@import '../../styles/variables.css';
+
+.button {
+	background-color: var(--button-bg);
+	border: 1px solid var(--button-bg);
+	box-shadow: inset 0 0 0 1px rgba(255, 255, 255, 0.25);
+	border-radius: 5px;
+	color: var(--button-color);
+	cursor: pointer;
+	font: var(--base-font);
+	font-weight: bold;
+	display: inline-block;
+	padding: 0.75em 1.25em;
+	transition: all 0.3s ease;
+}
+
+.button:hover,
+.button:focus,
+.button[aria-pressed="true"] {
+	background-color: var(--button-hover-bg);
+	border-color: var(--button-hover-bg);
+	outline: none;
+}
+
+.button[disabled],
+.button[disabled]:hover {
+	background-color: var(--button-disabled-bg);
+	border-color: var(--button-disabled-bg);
+	cursor: default;
+}
+
+.button[data-dojo-icon]:before {
```

Comment:
I was thinking more that whatever icon-font a user utilised, it would come with those kind of font size / weight etc... settings. Personally, I'd have expected a user to create an icon button by passing the icon through as part of the button text? Thus allowing any popular icon font to be used?
 ie. 
```
content: 'my icon button <i class="fa fa-save"></i>'
```